src/core/query/woql_compile.pl

```diff
@@ -1546,7 +1542,7 @@
     compile_arith(Arith,Pre_Term,ArithE).
 compile_wf(dot(Dict,Key,Value), get_dict(KeyE,DictE,ValueE)) -->
     resolve(Dict,DictE),
-    resolve(Key,KeyE),
+    {atom_string(KeyE,Key)},
```

Comment:
What is the difference between `{atom_string(KeyE,Key)}` and `atom_string(KeyE,Key)`?

Reply:
We are in a DCG so there are two implicit parameters unless we wrap in a `{ }` block